code/modules/species/outsider/vox.dm

```diff
@@ -151,6 +151,9 @@
 	damage_mask =     'icons/mob/human_races/species/vox/damage_mask_armalis.dmi'
 	blood_mask =      'icons/mob/human_races/species/vox/blood_mask_armalis.dmi'
 
+	speech_sounds = list('sound/voice/ashriek.ogg')
+	speech_chance = 10
+
```

Comment:
Might want to put this in the changelog just in case someone cares about it, other than that LGTM

Reply:
It is same with normal Vox, the only difference is that Armalis use ashriek instead.
But sure, can add into changelog.